lib/uktt.rb

```diff
@@ -1,20 +1,21 @@
 require 'uktt/version'
-require 'uktt/http'
-require 'uktt/section'
 require 'uktt/chapter'
-require 'uktt/heading'
-require 'uktt/country'
 require 'uktt/commodity'
+require 'uktt/country'
+require 'uktt/heading'
+require 'uktt/http'
 require 'uktt/monetary_exchange_rate'
-require 'uktt/quota'
+require 'uktt/parser'
 require 'uktt/pdf'
+require 'uktt/quota'
+require 'uktt/section'
 
 require 'yaml'
 require 'psych'
 
 module Uktt
-  API_HOST_PROD =       'https://www.trade-tariff.service.gov.uk/api'.freeze
-  API_HOST_LOCAL =      'http://localhost:3002/api'.freeze
+  API_HOST_PROD      = 'https://www.trade-tariff.service.gov.uk'.freeze
+  API_HOST_LOCAL     = 'https://dev.trade-tariff.service.gov.uk'.freeze
```

Comment:
So I don't know what we should do here. This client is written to integrate with a real api (:facepalm:). I end up making this change everytime I work on this repo to get the tests to pass. The alternative is having both the frontend and backend running everytime I want to essentially mock a real response

Reply:
I think the long-term goal should be to just mock out the real responses either using the faraday connection object or with contract testing

Reply:
Agreed!